HARK/ConsumptionSaving/ConsIndShockModel.py

```diff
@@ -3047,7 +3052,7 @@ def construct_assets_grid(parameters):
 
 # Make a dictionary to specify an infinite consumer with a four period cycle
 init_cyclical = copy(init_idiosyncratic_shocks)
-init_cyclical['PermGroFac'] = [1.082251, 2.8, 0.3, 1.1]
+init_cyclical['PermGroFac'] = [1.1, 1.082251, 2.8, 0.3]
```

Comment:
My only concern comes from my unfamiliarity with cyclical models in HARK.

For a finite horizon, non-cyclical model with two periods, I think HARK requires `len(PermGroFac) == 1`. The single element in the list is the growth factor of permanent income between the first and second periods.

How does this logic change when I'm considering an infinite-horizon model with a cycle of two periods? I know Chris, Matt and you had some lengthy discussion on how to treat the "terminal" period in cyclical vs non-cyclical models, which I did not follow, but I wanted to make sure this change (which looks good to me) is consistent with whatever you settled on.

Reply:
My understanding is that when `cycles == 0`, it's infinite horizon and so there is no terminal period solution.

When `cycles == N`, it's a finite model repeated many times. If a terminal solution is given, the solver returns N * T + 1 solutions.